docs/source/docker/advanced/code_examples/python_create_frame_predictions.py

```diff
@@ -0,0 +1,45 @@
+import av
+import json
+from pathlib import Path
+
+dataset_dir = Path('/datasets/my_dataset')
+video_path = dataset_dir / 'video.mp4'
+predictions_dir = dataset_dir / '.lightly' / 'predictions' / 'my_prediction_task'
+predictions_dir.mkdir(parents=True, exist_ok=True)
+
+def model_predict(frame):
+    # This function must be overwritten to generate predictions for a 
+    # frame using a prediction model of your choice.
+    return []
```

Comment:
I didn't add a typehint because the return type is `List[Dict[str, Any]]` which is honestly not very helpful. Instead I replaced the empty list with an example prediction. Hope that makes it clearer. The exact prediction format is explained in detail further up on the same page.

Reply:
Thanks for adding the example! 
I would at least add the typehint `List[Dict]`. It does not tell too much about the inner structure, but at least it tells a lot about which structures are all NOT allowed.